src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -7,16 +7,20 @@
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    using Orleans.Providers;
+
+    public class OrleansJsonSerializer : IExternalSerializer
```

Comment:
Users ought to be able to specify external serializers to use, even if only in certain cases.
This class could be used for storage providers, for example. Currently they use bits of this class (even if they were using it via `SerializationManager`).